sotodlib/site_pipeline/make_ml_map.py

```diff
@@ -112,7 +136,7 @@ def main(config_file=None, defaults=defaults, **args):
     prefix= args['odir'] + "/"
     if args['prefix']: prefix += args['prefix'] + "_"
     utils.mkdir(args['odir'])
-    L = mapmaking.init(level=mapmaking.DEBUG, rank=comm.rank)
+    L = mapmaking.log.init(level=mapmaking.log.DEBUG, rank=comm.rank)
```

Comment:
The problem here is that `mapmaking.init` is very difficult to understand that it creates a logger. I always think it creates a mapmaking object. There is a whole `log.py` file that includes everything. I think this is an interpreter mix up.

I want to keep the extra level since I think it improves readability. I'll fix it.

Reply:
This error is because `log.py` in mapmaking does `from logging import *`. I will fix it.